src/connectionmonitor/SignalingAndMetricsConnectionMonitor.ts

```diff
@@ -87,28 +87,28 @@ export default class SignalingAndMetricsConnectionMonitor
       this.connectionHealthData.setAudioSpeakerDelayMs(audioSpeakerDelayMs);
     }
 
-    // To get the total packets received, including RTCP, we need to sum up
+    // To get the total bytes received, including RTCP, we need to sum up
     // all candidate pair metrics (in case the candidate pair changes).
     //
     // The stats collector currently doesn't account for candidate pair stats
     // so we just use the raw report for now.
     const webrtcReport = clientMetricReport.getRTCStatsReport();
 
-    let totalPacketsReceived = 0;
+    let totalBytesReceived = 0;
     webrtcReport.forEach(stat => {
-      if (stat.type === 'candidate-pair' && stat.packetsReceived) {
-        totalPacketsReceived += stat.packetsReceived;
+      if (stat.type === 'candidate-pair' && stat.bytesReceived) {
```

Comment:
In the PR description, you mention Safari 17.3 or below so it never worked before since we introduced this code for safari right?
Can we mention in the PR description what the issue is (reconnect) and what NPM version we start to have this issue (3.20.0?)?

Reply:
Yes this was broken on arrival, will update description!